skbio/sequence/_dna.py

```diff
@@ -212,9 +217,14 @@ def transcribe(self):
         if self.has_positional_metadata():
             positional_metadata = self.positional_metadata
 
+        interval_metadata = None
```

Comment:
This method is marked as `stable`. These changes are not at the API level but can result in side effects. Just marking as I do not know if this is or is not a concern from the perspective of the API decorators